### DISCUSSION TITLE
I made a tool to convert AS3 sources into JS game

### DISCUSSION THREAD

**Discussion by GeneralVimes** [airsdk/Adobe-Runtime-Support#3135] [Show and tell]
I made a tool for myself and decided to make it public: https://github.com/GeneralVimes/AS32JS

I used it to convert my Farm and Mine AIR/Starling game (which is available for Android, iOS, Windows, Linux) into an HTML5 game.
Here is the AIR game: https://airapport.itch.io/farm-and-mine 
and here is the conversion result: https://poki.com/en/g/farm-and-mine

So, now my game is playable on the web, too. 

When I update my AS3 build I'm able to run the tool again and update the HML5 version as well.

**Comment by mrfrasier** [comment]
Very cool!
